homework-submission/class16-week1/src/components/Todolist.js

```diff
@@ -0,0 +1,34 @@
+import React, { Component } from "react";
+import Todo from "./Todo";
+
+class Todolist extends Component {
+  render() {
+    const list = [
```

Comment:
It is better to take this const out of the `render` function. But it before the class or in a standalone file.

Reply:
I moved the list const out of the class.